test/extension/config/default/extension_image_patch.yaml

```diff
@@ -6,5 +6,5 @@ spec:
   template:
     spec:
       containers:
-      - image: gcr.io/k8s-staging-cluster-api/test-extension:main
```

Comment:
This diff should not be commited (same for pull policy yaml)

If you want, let's create an issue so we can potentially get rid of those "source updates" when running e2e tests.